MPF.Modules/BaseParameters.cs

```diff
@@ -1849,9 +1849,8 @@ protected static string GetPlayStation5Version(char? driveLetter)
                     case 'A': return Region.Asia;
                     case 'C': return Region.China;
                     case 'E': return Region.Europe;
-                    case 'J': return Region.JapanKorea;
```

Comment:
I disagree with this change. Users are more than capable of selecting the region that their particular disc is from. There's a reason it's both JP/KR as it is now.